html/browsers/windows/browsing-context-names/browsing-context-_blank.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML Test: Browsing context - `_blank` name keyword</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+test(t => {
+  var window1 = window.open('about:blank', '_blank');
+  var window2 = window.open('about:blank', '_blank');
+  var window3 = window.open('about:blank', '_blank');
+  t.add_cleanup(() => {
+    window1.close();
+    window2.close();
+    window3.close();
+  });
+  assert_false(window1 === window2);
+  assert_false(window2 === window3);
+  assert_false(window1 === window3);
+}, 'window.open into `_blank` should create a new browsing context each time');
+
+test(t => {
```

Comment:
Test failures in Safari/Webkit and Chrome here for case-insensitive `_blank` appear buggy to me (spec asserts these keywords should be case-insensitive), but I think there is a good chance that bugginess could extend to other keywords (`_top`, `_parent`, etc.) so I'll hold back on opening bugs for the moment.